core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -327,6 +327,7 @@ object Defaults {
   val LiAsyncFetcherEnabled = false
   val LiNumControllerInitThreads = 1
   val LiLogCleanerFineGrainedLockEnabled = true
+  val LiDropCorruptedFilesEnabled = false
```

Comment:
(Non-blocking) Do we want to separate the config for dropping log segments & checkpoint files?

Reply:
For now, my thinking is that we don't need to have the separation.